Classification/cnns/ofrecord_util.py

```diff
@@ -90,14 +90,19 @@ def load_imagenet_for_training(args):
                                         part_name_suffix_length=5,
                                         random_shuffle=True,
                                         shuffle_after_epoch=True)
-    image = flow.data.OFRecordImageDecoderRandomCrop(ofrecord, "encoded",  # seed=seed,
```

Comment:
suggest to define a new `load_imagenet_for_training` for gpu jpeg decoder, keep old one(cpu jpeg decoder) for fp32 training.